items/inventory.go

```diff
@@ -0,0 +1,14 @@
+package items
+
+type Inventory struct {
+	items []Item `json:"items"`
```

Comment:
so i'm still confused about why we're storing the entire Item object. I don't think this is a good idea because it'll make it harder to update items across the entire playerbase.

Reply:
Mhmm , what do you suggest?

Reply:
Well I would imagine that we would reference the item's Key, and that would then reference the struct. Now that I think about it, it might be prudent to store the item struct as part of the player so they could modify it etc.